src/material-experimental/mdc-form-field/form-field.html

```diff
@@ -0,0 +1,69 @@
+<ng-template #labelTemplate>
+  <!-- We add aria-owns as a workaround for an issue in JAWS & NVDA where the label isn't
+     read if it comes before the control in the DOM. -->
+  <label matFormFieldFloatingLabel [floating]="_shouldLabelFloat()"
+         *ngIf="_hasFloatingLabel()"
+         (cdkObserveContent)="_rerenderOutlineNotch()"
+         [cdkObserveContentDisabled]="!_hasOutline()"
+         [id]="_labelId"
+         [attr.for]="_control.id"
+         [attr.aria-owns]="_control.id">
+    <ng-content select="mat-label"></ng-content>
+
+    <!-- Manually handle the required asterisk. This is necessary because MDC can only
+         display the asterisk if the label is directly preceded by the input. This cannot
+         be guaranteed here since the form control is not necessarily an input, or is wrapped.
+      -->
+    <span class="mat-mdc-form-field-required-marker" aria-hidden="true"
+          *ngIf="!hideRequiredMarker && _control.required && !_control.disabled">&#32;*</span>
+  </label>
+</ng-template>
```

Comment:
I don't think so. Since there is not always a floating label, we need to have an `*ngIf` that already wraps the `<ng-content>` for the label with an `ng-template`.

We might be able to get rid of the first `ng-template` if the prefix/suffix thing for the outline appearance is solved.. but it will still require the `*ngIf` on the `<label>` element.

Reply:
This is the same in the current form-field implementation, and I'd also say that it is rare that directives are used in the label. I'm wondering if Angular Ivy somehow fixed that double-instantiation? I remember reading something but I'm not too sure.